server/utils/ssotToRobotParsing/__tests__/SsotToRobotParser.test.js

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable no-undef */
-const parser = require('../SsotToRobotParser');
+const parser = require('../ssotToRobotParser');
 const testSsot = require('./SsotForTesting.json');
-const dbHandler = require('../../../utils/TestingUtils/TestDatabaseHandler');
-const dbLoader = require('../../../utils/TestingUtils/databaseLoader');
+const dbHandler = require('../../testingUtils/testDatabaseHandler');
+const dbLoader = require('../../testingUtils/databaseLoader');
 const {
   testRobotId,
   testJobId,
   testRobotCode,
-} = require('../../../utils/TestingUtils/testData');
+} = require('../../testingUtils/testData');
```

Comment:
I just realised here that I would rename the folder to just testing or something simular. I think "utils" should not be part of the folder name

Reply:
i've renamed it to `testing`